app/design/adminhtml/default/default/template/ebizmarts/mailchimp/sales/order/view/monkey.phtml

```diff
@@ -1,7 +1,7 @@
 <?php if($this->isReferred()) : ?>
 <div class="entry-edit box-right">
     <div class="entry-edit-head">
-        <h4 class="icon-head"><?php echo $this->quoteEscape($this->__('MailChimp')) ?></h4>
+        <h4 class="icon-head"><?php echo $this->getMailChimpHelper()->mcEscapeQuote($this->__('MailChimp')) ?></h4>
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.